tests/apps/dial/dial_progress/configs/ast1/extensions.conf

```diff
@@ -4,8 +4,8 @@ exten => s,1,Answer()
 	same => n,UserEvent(DialProgress,Result: Pass)
 	same => n,Dial(Local/s@earlymedia,3^2) ; we should get early media, so that will "lock" in this call and we'll wait up to 3 seconds, resulting in a normal NOANSWER
 	same => n,ExecIf($["${DIALSTATUS}"="NOANSWER"]?UserEvent(DialProgress,Result: Pass))
-	same => n,Dial(Local/s@answer,3^2) ; call will answer, so should not continue
-	same => n,UserEvent(DialProgress,Result: Fail)
+	same => n,Dial(Local/s@answer,3^2,,g) ; call will answer, and will continue
```

Comment:
You have 2 commas so the `g` isn't being picked up as an option which means the call won't continue to the next priority after hangup.